packages/graphql/src/types/output/Itinerary.js

```diff
@@ -11,6 +11,11 @@ export default new GraphQLObjectType({
   name: 'Itinerary',
   fields: {
     id: GlobalID(({ id }) => id),
+    price: { type: GraphQLInt },
+    flyFrom: { type: GraphQLString },
+    flyTo: { type: GraphQLString },
+    localDeparture: { type: GraphQLString },
+    localArrival: { type: GraphQLString },
```

Comment:
maybe we want some reusable type like 
[RouteStop](https://gitlab.skypicker.com/graphql/graphql/blob/master/src/apps/flight/types/outputs/RouteStop.js)

Reply:
We handle this as the separate issue.